src/Webform.js

```diff
@@ -1135,7 +1135,7 @@ export default class Webform extends NestedComponent {
    */
   submit(before, options) {
     if (!before) {
-      return this.beforeSubmit(options).then(() => this.executeSubmit(options));
+      return this.beforeSubmit(options).then(() => this.executeSubmit(options)).catch(console.log);
```

Comment:
Calling ```catch(console.log)``` is pretty dangerous since we don't know if the ```this``` pointer would ever be used within the console object. If so, then your call here would crash since ```this``` would be pointing to the window object. 

This should probably say ```catch(err => console.log(err))``` or actually, we would probably prefer to use ```console.warn``` so change it to this.

```js
catch(err => console.warn(err));
```